src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,224 @@
+
+<p align="center"><Image src="/docs/cloud/images/external_resources_sqs_demo.png" alt="AWS SQS external resource demo UI component" width="450" /></p>
+
+## Example: Deploy a Development Environment with External Resources
+
+Now that we've walked through a sample of an `external` configuration, let's look at using this within the broader context of an application we're working on. You can navigate to our [demo repo](https://github.com/okteto/external-resources-aws) to follow along if you'd like.
+
+In this example we'll be configuring and deploying a development environment that includes polyglot microservices, an AWS SQS queue, and an S3 bucket.
+
+<p align="center"><Image src="/docs/cloud/images/external_resources_architecture.png" alt="Application architecture diagram" width="450" /></p>
+
+### Prerequisites
+
+1. Okteto CLI 2.14 or newer
+1. An AWS account
+1. An Okteto account
+1. Create a set of IAM keys for your AWS account (If you are using Okteto Self-Hosted, you can directly assign an AWS Role)
+1. Create the following Okteto secrets:
+
+        AWS_ACCESS_KEY_ID: The Acces Key ID of your IAM user
+        AWS_SECRET_ACCESS_KEY: The Secret Acces Key of your IAM user
+        AWS_REGION: The region in AWS you would like to use for the external resources
+
+Notice the `external` configuration in this demo. It specifies most attributes except for `url` for most of these resources.
+
+```yaml
+external:
+  readme:
+    icon: okteto
+    notes: README.md
+    endpoints:
+    - name: readme
+      url: https://github.com/okteto/external-resources-aws
+  sqs:
+    icon: aws
+    notes: sqs/notes.md
+    endpoints:
+    - name: queue
+  s3:
+    icon: aws
+    notes: s3/notes.md
+    endpoints:
+    - name: bucket
+  api-docs:
+    icon: dashboard
+    notes: check/notes.md
+    endpoints:
+    - name: docs
+```
+
+The `url:` keys are missing from most external resource (`sqs`, `s3`, and `api-docs`) because they've been defined in the `deploy` > `commands` > `command` section using the [Okteto Secrets](/cloud/secrets) you defined [above](#prerequisites) and the [dynamic endpoints](#using-dynamic-endpoints-with-external-resources) feature.
+
+Alternatively, you could remove the respective `deploy` commands and add the `url` endpoints here manually, but this is a good example of how flexible the manifest file can be.
+
+Once you've configured the prerequisites, you can deploy this on your Okteto instance:
+
+```
+$ git clone https://github.com/okteto/external-resources-aws
+$ cd external-resources-aws
+$ okteto context use $OKTETO_URL
+$ okteto deploy
+```
+
+Once the application has been deployed in Okteto, you can work on the `menu` microservice by running the command: `okteto up menu`. In this specific example, once you've executed this command you'll also need to run `yarn start` from the container.
```

Comment:
Can we add a paragraph iterating that the command above will deploy the environment and the cloud resources and all the things?